server/src/main/java/com/linecorp/centraldogma/server/internal/mirror/GitMirror.java

```diff
@@ -395,6 +398,16 @@ private GitWithAuth openGit(File workDir) throws Exception {
         }
     }
 
+    private static String generateCommitDetail(RevCommit headCommit) {
+        final String newLine = "\n";
+        final PersonIdent authorIdent = headCommit.getAuthorIdent();
+        return "The commit information of " + ": " + newLine +
+               "Author: " + authorIdent.getName() + " <" +
+               authorIdent.getEmailAddress() + "> " + newLine +
+               "Date: " + authorIdent.getWhen() + newLine +
+               "Message: " + headCommit.getFullMessage();
```

Comment:
- I think we can just use the new line character.
- How about adding another new line before the message and removing `Message: `?
```suggestion
        final PersonIdent authorIdent = headCommit.getAuthorIdent();
        return "The commit information" + '\n' +
               "Author: " + authorIdent.getName() + " <" + authorIdent.getEmailAddress() + "> " + '\n' +
               "Date:   " + authorIdent.getWhen() + '\n' + '\n' +
               headCommit.getFullMessage();
```

Reply:
@minwoox I agree with removing `Message` but why two new lines? 🤔
The result will be
```
    Author: ...
    Date: ...

    <Message>

Reply:
Ah I see the template you suggest when I try `git log` ☺️

Reply:
I was inspired by git log. 😄 
<img width="708" alt="image" src="https://user-images.githubusercontent.com/25103250/225551857-9815922c-87e8-47dc-95e5-e846ce085869.png">

Reply:
Yes, I see, thanks.